Source/UI/NewUI/PaymentForm/ViewController/PaymentViewController.swift

```diff
@@ -16,7 +16,8 @@ final class PaymentViewController: UIViewController {
 
   private(set) var viewModel: PaymentViewModel
   private var notificationCenter = NotificationCenter.default
-
+  private var currentOrientation: UIDeviceOrientation = .portrait
+  private var oldOrientation: UIDeviceOrientation = .portrait
```

Comment:
What is the purpose of these properties?

Reply:
when payment view controller did disappeared then return back the the old stored orientation

Reply:
If you forced orientation change on the device there is a high chance the user has turned the device around and hasn't been twisting their head the whole time 😬. When you finish you tell user to turn device as they were before?

Reply:
We are gonna support landscape soon.

Reply:
I don't think you followed me. Imagine this:
User is interacting with their app as they usually do, relaxing in landscape mode.
Their app takes them to our payment form, which turns their app into portrait mode.
At this stage the user is most likely going to turn their device around and stop relaxing in landscape, being more active in portrait.
They happily complete the smooth payment journey we prepared for them and the screen is dismissed
🟡 We now tell turn device orientation again for user and tell them to go back to relaxing in landscape
❗ What if user wanted to continue being active in portrait and weren't ready to resume relaxing? Why we tell user what to do?

Reply:
https://user-images.githubusercontent.com/102028170/182799693-91a40669-ca8b-4e50-857a-c0c82cd996b3.mov

Reply:
yeah i think we should just leave them in portrait, its already bad enough that we're forcing them to switch orientations once